### ISSUE TITLE
mock.stopAll()

### ISSUE THREAD

**Issue by mingos777** [boblauer/mock-require#2]
It would be nice if there was a possibility to stop mocking all modules with a single call:
```
mock.stopAll();
```
or similar.

The rationale behind my request is that I often have multiple mocks in dozens of test cases and it's annoyingly repetitive to add the `afterEach` in every single test case with all the mocks I have created. Instead, it would be much cleaner to just create a single `afterEach` call after every test in the entire test suite (Jasmine allows this), reducing greatly the amount of boilerplate code:
```
var mock = require("mock-require");
afterEach(function() {
    mock.stopAll();
});
```

I can provide a pull request if needed.

**Comment by boblauer** [comment]
Great idea.  A pull request would be much appreciated.

**Comment by mingos777** [comment]
OK, in ~2 hours' time I believe I can manage to find some free time, so I'll try to come up with a PR.

**Comment by mingos777** [comment]
Pull request opened (df02000).

**Comment by boblauer** [comment]
Thanks for the pull!  Published on npm at `1.1.0`.

**Comment by mingos777** [comment]
And thanks for publishing the new version! Works like a charm and my code suddenly got [cleaner](https://bitbucket.org/mingos/ucms.js/commits/80f0e844fff87b421dad503ae3c14da3938676e5) :).
